### DISCUSSION TITLE
partnersuccess.cisco.com

### DISCUSSION THREAD

**Discussion by hwprinz** [StevenBlack/hosts#1601] [General]
Hi, 
I'm myself a Cisco Partner, with my small Company. As a partner, i need to go every now and then, as the name implicits, to partnersuccess.cisco.com website. Since a while it's getting blocked through your host list by pihole's gravity...
so, i would need to understand what's on a regular partner website of a manufacturer to be blocked..
Even if there is the possibility to whitelist it, it makes no sense.... i'm really confused about this classification.
Thanks

**Comment by StevenBlack** [comment]
Hello @hwprinz.  Why is this filed in discussions?  Next time please make an issue, not a discussion.  🙄  This is not going to be a discussion.

The domain `partnersuccess.cisco.com` comes to us via Adguard-cname hosts.  Please report this issue [here](https://github.com/AdguardTeam/cname-trackers/issues).
